Documentation/bSDD API.md

```diff
@@ -31,7 +29,7 @@ Don’t forget to check the “read” scope!
 
 ## Using https://identifier.buildingsmart.org
 !! For system to system communication using these identifier URIs is not recommended. !!
-You can access the data of classification or property also directly via the URI of the classification or property. For example, you can navigate in the browser to https://identifier.buildingsmart.org/uri/buildingsmart/ifc/4.3/class/IfcWall and then you will see a visual representation of the data of that classification. If you would like the output in json format, then sending an "Accept" header with "application/json" will give you a result in json. The content of this json result differs from the html result!
+You can access the data of Class or property also directly via the URI of the Class or property. For example, you can navigate in the browser to https://identifier.buildingsmart.org/uri/buildingsmart/ifc/4.3/class/IfcWall and then you will see a visual representation of the data of that Class. If you would like the output in json format, then sending an "Accept" header with "application/json" will give you a result in json. The content of this json result differs from the html result!
```

Comment:
This renaming makes sense, but could you @Baars-IT by the way explain this paragraph better? For example, provide clear guidance on what should be done instead.